locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,57 @@
+# links
+-link-firefox-password-manager = <a href="https://www.mozilla.org/en-US/firefox/features/password-manager/">Firefox Password Manager</a>
```

Comment:
These URLs are not supposed to be translated, so these links should not be defined as terms.

Also, there is a difference between a term and a message, and both can be referenced in other message. This is not a good use for terms (brand name typically is).

Also, remove `en-US` from mozilla.org strings, you should let the browser ask for the right language.

Reply:
Is there a way to reference the same string in only this local file (local variable) in fluent?
Are you suggesting that we duplicate this string itself in multiple places in the file?

Reply:
You should put the text in the string, even if that means duplicating some content.

As mentioned, the URL should not be exposed in the string.